components/rmrk/Gallery/Holder/Holder.vue

```diff
@@ -100,11 +100,22 @@
           <b-table-column
             :visible="columnsVisible['Sale'].display"
             field="Sale"
-            label="Sale"
+            :label="saleHeaderLabel"
             sortable
             v-slot="props">
             {{ props.row.SaleFormatted }}
           </b-table-column>
+          <b-table-column
```

Comment:
DRY

Reply:
I agree with DRY, but in this case, the html part seems need to be customised column by column. 

They always have something to specialize
For example:

```html
    <td
      v-show="columnsVisible['Amount'].display"
      class="has-text-right">
      {{ item.Amount }}
    </td>
    <td v-show="columnsVisible['Bought'].display">
      {{ item.BoughtFormatted }}
    </td>
```


```html
          <b-table-column
            v-if="groupKey === 'Flipper'"
            :visible="columnsVisible['Percentage'].display"
            field="Percentage"
            label="Percentage"
            sortable
            v-slot="props">
            <span :class="percentageTextClassName(props.row.Percentage)">
              {{ props.row.Percentage | toPercent('-') }}
            </span>
          </b-table-column>

```